aiocogeo/tiler.py

```diff
@@ -54,7 +65,7 @@ def __iter__(self):
         Allow for variable expansion (``arr, mask = TileResponse``)
 
         """
-        for i in (self.arr, self.mask):
+        for i in (self.arr, self.mask * 255):
```

Comment:
there is still some work to do for this class to be utilized

Reply:
I'm fine not including it for now.  Is there a plan to eventually support something similar in rio-tiler? (similar to https://github.com/cogeotiff/rio-tiler/issues/296 but that issue is scoped to just metadata/info so not sure).